modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/rest-openapi.yaml

```diff
@@ -678,6 +678,9 @@ components:
                     type: integer
                 description:
                     type: string
+                externalReferenceCode:
```

Comment:
becase this will be the option externalReferenceCode let's call it optionExternalReferenceCode (like the optionId) below